src/screens/AddCashSheet.js

```diff
@@ -90,6 +98,15 @@ export default function AddCashSheet() {
     [stopErrorTimeout, cashLimits, startErrorTimeout, onClearError]
   );
 
+  useEffect(() => {
+    if (wyreAuthenticationUrl) {
+      setWyreAuthenticationFailureTimeout(
+        wyreAuthenticationFlowFailureCallback,
+        10_000
+      );
+    }
```

Comment:
Once we have a authUrl, we give it 10 seconds to complete. Guessing this isn't going to work though, since the user may need to complete a 3DS challenge 🤔